index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">
+    <div class="wrapper">
       <div class="nav-left">
-        <a href="#" class="logo">BankPL</a>
-        <p class="offer-name">Konto 360°</p>
+        <a href="#" class="site-logo">BankPL</a>
+        <span class="offer-name">Konto 360°</p>
       </div>
-      <a href="#" class="nav-icon">&#9776;</a>
+      
+      <button href="#" class="nav-icon">&#9776;</button>
+
       <ul class="main-nav">
-        <li><a href="#">O koncie</a></li>
-        <li><a href="#">Pakiet Bardzo Pomocny</a></li>
-        <li><a href="#">Lubię to polecam</a></li>
-        <li><a href="#">Jak otworzyć</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">O koncie</a></li>

```

Comment:
wprowadzenie do BEM - ogolnie o tym wiecej w necie, ja wyjasnie co nam to daje w tym przypadku - plaska strukture styli - latwiejsza do nadpisania, i szybsza dla przegladarki
mniej kodu nie znaczy lepiej - tutaj troche wiecej do pisania, ale latwiej do zarzadzania 

```
   .main-nav { display: block; }
   .main-nav__item { display: inline-block; }
   .main-nav__link { color: red; display: block; padding: 6px 16px; }
```

i teraz jesli chcemy dodac nowy wyglad dla jakiegos item - np wide, albo special, albo cokolwiek innego - to nie musimy pisac calego dlugiego selektora zagniezdzenia - co zwalszcza jest upierdliwe przy kolejnym podmenu i ich nastepnych podmenu. 
po prostu dodamy kolejny selektor modyfikujacy:
`.main-nav__item--wide { min-width: 300px; }` oraz 
`.main-nav__item--special { border: 1px solid green; }` 


W sumie do samego `<ul>` mozna by dodac klase `.menu` - ktora jedyne co robi, to:
```
.menu {
list-style-type: none;
margin: 0;
display: block;
padding: 0;
}
```
i we wszystkich menu na stronie, uzyjecie sobie tej klasy do zresetowania listowania